mne/bem.py

```diff
@@ -1355,3 +1357,260 @@ def write_bem_solution(fname, bem):
                                bem['solution'])
         end_block(fid, FIFF.FIFFB_BEM)
         end_file(fid)
+
+
+# #############################################################################
+# Create 3-Layers BEM model from Flash MRI images
+
+def make_flash_bem(subject, subjects_dir, noflash30=False, noconvert=False,
+                   unwarp=False, show=False):
+    """Create 3-Layers BEM model from Flash MRI images
+
+    Parameters
+    -----------
+    subject : string
+        Subject name.
+    subjects_dir : string
+        Directory containing subjects data (Freesurfer SUBJECTS_DIR).
+    noflash30 : bool
+        Skip the 30-degree flip angle data.
+    noconvert : bool
```

Comment:
maybe we can pythonize them here at the function level but expose the original names at the command line to keep backwards compatibility. cc@agramfort